packages/foreman/tfm/tfm.spec

```diff
@@ -103,6 +103,7 @@ Obsoletes: %{scl_prefix}rubygem-smart_proxy_abrt < 0.0.8-4
 Obsoletes: %{scl_prefix}rubygem-sprockets < 3.5.2-2
 Obsoletes: %{scl_prefix}rubygem-sprockets-rails < 2.3.3-2
 Obsoletes: %{scl_prefix}rubygem-table_print < 1.5.1-7
+Obsoletes: %{scl_prefix}rubygem-tilt < 2.0.8-4
```

Comment:
I think this should be done in rubygem-sass-rails that actually obsoleted this gem